src/core_modules/capture-core/components/FormFields/New/Fields/OrgUnitField/SingleOrgUnitSelectField.component.js

```diff
@@ -67,6 +77,7 @@ class SingleOrgUnitSelectFieldPlain extends React.Component<Props> {
 
     onDeselectOrgUnit = () => {
         this.props.onBlur(null);
+        this.props.value && this.setState({ previousOrgUnitId: this.props.value.id });
```

Comment:
Hey, 

Maybe the [lgtm alert](https://lgtm.com/projects/g/dhis2/capture-app/rev/pr-d281e681b7d6f46cf589008700fbd38045a79a5f) is fixed if `this.setState` is called first? Like this:
```
onDeselectOrgUnit = () => {
        this.props.value && this.setState({ previousOrgUnitId: this.props.value.id });
        this.props.onBlur(null);
}
```
Can you have a look? Thanks!